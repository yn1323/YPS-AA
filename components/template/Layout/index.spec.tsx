import React from 'react'
import Component from '.'
import { mountWithTheme } from '@test/TestWrapper'

test(`${Component.name}-snapshot`, () => {
  const wrapper = mountWithTheme(
    <Component>
      <div>hoge</div>
    </Component>
  )
  expect(wrapper).toMatchSnapshot()
})
