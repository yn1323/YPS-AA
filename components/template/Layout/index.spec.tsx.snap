// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout-snapshot 1`] = `
<Layout>
  <Layout__Container>
    <div
      className="Layout__Container-sc-1mgn808-0 fJDiXB"
    >
      <Header>
        <WithStyles(ForwardRef(AppBar))
          position="fixed"
        >
          <ForwardRef(AppBar)
            classes={
              Object {
                "colorDefault": "MuiAppBar-colorDefault",
                "colorInherit": "MuiAppBar-colorInherit",
                "colorPrimary": "MuiAppBar-colorPrimary",
                "colorSecondary": "MuiAppBar-colorSecondary",
                "colorTransparent": "MuiAppBar-colorTransparent",
                "positionAbsolute": "MuiAppBar-positionAbsolute",
                "positionFixed": "MuiAppBar-positionFixed",
                "positionRelative": "MuiAppBar-positionRelative",
                "positionStatic": "MuiAppBar-positionStatic",
                "positionSticky": "MuiAppBar-positionSticky",
                "root": "MuiAppBar-root",
              }
            }
            position="fixed"
          >
            <WithStyles(ForwardRef(Paper))
              className="MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed"
              component="header"
              elevation={4}
              square={true}
            >
              <ForwardRef(Paper)
                className="MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed"
                classes={
                  Object {
                    "elevation0": "MuiPaper-elevation0",
                    "elevation1": "MuiPaper-elevation1",
                    "elevation10": "MuiPaper-elevation10",
                    "elevation11": "MuiPaper-elevation11",
                    "elevation12": "MuiPaper-elevation12",
                    "elevation13": "MuiPaper-elevation13",
                    "elevation14": "MuiPaper-elevation14",
                    "elevation15": "MuiPaper-elevation15",
                    "elevation16": "MuiPaper-elevation16",
                    "elevation17": "MuiPaper-elevation17",
                    "elevation18": "MuiPaper-elevation18",
                    "elevation19": "MuiPaper-elevation19",
                    "elevation2": "MuiPaper-elevation2",
                    "elevation20": "MuiPaper-elevation20",
                    "elevation21": "MuiPaper-elevation21",
                    "elevation22": "MuiPaper-elevation22",
                    "elevation23": "MuiPaper-elevation23",
                    "elevation24": "MuiPaper-elevation24",
                    "elevation3": "MuiPaper-elevation3",
                    "elevation4": "MuiPaper-elevation4",
                    "elevation5": "MuiPaper-elevation5",
                    "elevation6": "MuiPaper-elevation6",
                    "elevation7": "MuiPaper-elevation7",
                    "elevation8": "MuiPaper-elevation8",
                    "elevation9": "MuiPaper-elevation9",
                    "outlined": "MuiPaper-outlined",
                    "root": "MuiPaper-root",
                    "rounded": "MuiPaper-rounded",
                  }
                }
                component="header"
                elevation={4}
                square={true}
              >
                <header
                  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary mui-fixed MuiPaper-elevation4"
                >
                  <Header__SpacedToolbar
                    $isLoggedIn={false}
                    variant="dense"
                  >
                    <WithStyles(ForwardRef(Toolbar))
                      className="Header__SpacedToolbar-sc-1u3zo5s-0 eYneNE"
                      variant="dense"
                    >
                      <ForwardRef(Toolbar)
                        className="Header__SpacedToolbar-sc-1u3zo5s-0 eYneNE"
                        classes={
                          Object {
                            "dense": "MuiToolbar-dense",
                            "gutters": "MuiToolbar-gutters",
                            "regular": "MuiToolbar-regular",
                            "root": "MuiToolbar-root",
                          }
                        }
                        variant="dense"
                      >
                        <div
                          className="MuiToolbar-root MuiToolbar-dense Header__SpacedToolbar-sc-1u3zo5s-0 eYneNE MuiToolbar-gutters"
                        >
                          <HeaderTitle>
                            <HeaderTitle__ImageLogo
                              height={32}
                              src="/images/logo.png"
                              width={32}
                            >
                              <Image1
                                className="HeaderTitle__ImageLogo-sc-nwr99g-1 gqIWiZ"
                                height={32}
                                src="/images/logo.png"
                                width={32}
                              >
                                <div
                                  style={
                                    Object {
                                      "boxSizing": "border-box",
                                      "display": "inline-block",
                                      "margin": 0,
                                      "maxWidth": "100%",
                                      "overflow": "hidden",
                                      "position": "relative",
                                    }
                                  }
                                >
                                  <div
                                    style={
                                      Object {
                                        "boxSizing": "border-box",
                                        "display": "block",
                                        "maxWidth": "100%",
                                      }
                                    }
                                  >
                                    <img
                                      alt=""
                                      aria-hidden={true}
                                      src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzIiIGhlaWdodD0iMzIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                                      style={
                                        Object {
                                          "border": "none",
                                          "display": "block",
                                          "margin": 0,
                                          "maxWidth": "100%",
                                          "padding": 0,
                                        }
                                      }
                                    />
                                  </div>
                                  <noscript>
                                    <img
                                      className="HeaderTitle__ImageLogo-sc-nwr99g-1 gqIWiZ"
                                      data-nimg={true}
                                      decoding="async"
                                      src="/_next/image?url=%2Fimages%2Flogo.png&w=64&q=75"
                                      srcSet="/_next/image?url=%2Fimages%2Flogo.png&w=32&q=75 1x, /_next/image?url=%2Fimages%2Flogo.png&w=64&q=75 2x"
                                      style={
                                        Object {
                                          "border": "none",
                                          "bottom": 0,
                                          "boxSizing": "border-box",
                                          "display": "block",
                                          "height": 0,
                                          "left": 0,
                                          "margin": "auto",
                                          "maxHeight": "100%",
                                          "maxWidth": "100%",
                                          "minHeight": "100%",
                                          "minWidth": "100%",
                                          "objectFit": undefined,
                                          "objectPosition": undefined,
                                          "padding": 0,
                                          "position": "absolute",
                                          "right": 0,
                                          "top": 0,
                                          "width": 0,
                                        }
                                      }
                                    />
                                  </noscript>
                                  <img
                                    className="HeaderTitle__ImageLogo-sc-nwr99g-1 gqIWiZ"
                                    data-nimg={true}
                                    decoding="async"
                                    src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                                    style={
                                      Object {
                                        "border": "none",
                                        "bottom": 0,
                                        "boxSizing": "border-box",
                                        "display": "block",
                                        "height": 0,
                                        "left": 0,
                                        "margin": "auto",
                                        "maxHeight": "100%",
                                        "maxWidth": "100%",
                                        "minHeight": "100%",
                                        "minWidth": "100%",
                                        "objectFit": undefined,
                                        "objectPosition": undefined,
                                        "padding": 0,
                                        "position": "absolute",
                                        "right": 0,
                                        "top": 0,
                                        "width": 0,
                                      }
                                    }
                                  />
                                </div>
                              </Image1>
                            </HeaderTitle__ImageLogo>
                            <HeaderTitle__TitleText
                              variant="h6"
                            >
                              <WithStyles(ForwardRef(Typography))
                                className="HeaderTitle__TitleText-sc-nwr99g-0 fxMhyC"
                                variant="h6"
                              >
                                <ForwardRef(Typography)
                                  className="HeaderTitle__TitleText-sc-nwr99g-0 fxMhyC"
                                  classes={
                                    Object {
                                      "alignCenter": "MuiTypography-alignCenter",
                                      "alignJustify": "MuiTypography-alignJustify",
                                      "alignLeft": "MuiTypography-alignLeft",
                                      "alignRight": "MuiTypography-alignRight",
                                      "body1": "MuiTypography-body1",
                                      "body2": "MuiTypography-body2",
                                      "button": "MuiTypography-button",
                                      "caption": "MuiTypography-caption",
                                      "colorError": "MuiTypography-colorError",
                                      "colorInherit": "MuiTypography-colorInherit",
                                      "colorPrimary": "MuiTypography-colorPrimary",
                                      "colorSecondary": "MuiTypography-colorSecondary",
                                      "colorTextPrimary": "MuiTypography-colorTextPrimary",
                                      "colorTextSecondary": "MuiTypography-colorTextSecondary",
                                      "displayBlock": "MuiTypography-displayBlock",
                                      "displayInline": "MuiTypography-displayInline",
                                      "gutterBottom": "MuiTypography-gutterBottom",
                                      "h1": "MuiTypography-h1",
                                      "h2": "MuiTypography-h2",
                                      "h3": "MuiTypography-h3",
                                      "h4": "MuiTypography-h4",
                                      "h5": "MuiTypography-h5",
                                      "h6": "MuiTypography-h6",
                                      "noWrap": "MuiTypography-noWrap",
                                      "overline": "MuiTypography-overline",
                                      "paragraph": "MuiTypography-paragraph",
                                      "root": "MuiTypography-root",
                                      "srOnly": "MuiTypography-srOnly",
                                      "subtitle1": "MuiTypography-subtitle1",
                                      "subtitle2": "MuiTypography-subtitle2",
                                    }
                                  }
                                  variant="h6"
                                >
                                  <h6
                                    className="MuiTypography-root HeaderTitle__TitleText-sc-nwr99g-0 fxMhyC MuiTypography-h6"
                                  >
                                    YPS
                                  </h6>
                                </ForwardRef(Typography)>
                              </WithStyles(ForwardRef(Typography))>
                            </HeaderTitle__TitleText>
                          </HeaderTitle>
                          <WithStyles(ForwardRef(Button))
                            color="primary"
                            variant="contained"
                          >
                            <ForwardRef(Button)
                              classes={
                                Object {
                                  "colorInherit": "MuiButton-colorInherit",
                                  "contained": "MuiButton-contained",
                                  "containedPrimary": "MuiButton-containedPrimary",
                                  "containedSecondary": "MuiButton-containedSecondary",
                                  "containedSizeLarge": "MuiButton-containedSizeLarge",
                                  "containedSizeSmall": "MuiButton-containedSizeSmall",
                                  "disableElevation": "MuiButton-disableElevation",
                                  "disabled": "Mui-disabled",
                                  "endIcon": "MuiButton-endIcon",
                                  "focusVisible": "Mui-focusVisible",
                                  "fullWidth": "MuiButton-fullWidth",
                                  "iconSizeLarge": "MuiButton-iconSizeLarge",
                                  "iconSizeMedium": "MuiButton-iconSizeMedium",
                                  "iconSizeSmall": "MuiButton-iconSizeSmall",
                                  "label": "MuiButton-label",
                                  "outlined": "MuiButton-outlined",
                                  "outlinedPrimary": "MuiButton-outlinedPrimary",
                                  "outlinedSecondary": "MuiButton-outlinedSecondary",
                                  "outlinedSizeLarge": "MuiButton-outlinedSizeLarge",
                                  "outlinedSizeSmall": "MuiButton-outlinedSizeSmall",
                                  "root": "MuiButton-root",
                                  "sizeLarge": "MuiButton-sizeLarge",
                                  "sizeSmall": "MuiButton-sizeSmall",
                                  "startIcon": "MuiButton-startIcon",
                                  "text": "MuiButton-text",
                                  "textPrimary": "MuiButton-textPrimary",
                                  "textSecondary": "MuiButton-textSecondary",
                                  "textSizeLarge": "MuiButton-textSizeLarge",
                                  "textSizeSmall": "MuiButton-textSizeSmall",
                                }
                              }
                              color="primary"
                              variant="contained"
                            >
                              <WithStyles(ForwardRef(ButtonBase))
                                className="MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                                component="button"
                                disabled={false}
                                focusRipple={true}
                                focusVisibleClassName="Mui-focusVisible"
                                type="button"
                              >
                                <ForwardRef(ButtonBase)
                                  className="MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                                  classes={
                                    Object {
                                      "disabled": "Mui-disabled",
                                      "focusVisible": "Mui-focusVisible",
                                      "root": "MuiButtonBase-root",
                                    }
                                  }
                                  component="button"
                                  disabled={false}
                                  focusRipple={true}
                                  focusVisibleClassName="Mui-focusVisible"
                                  type="button"
                                >
                                  <button
                                    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                                    disabled={false}
                                    onBlur={[Function]}
                                    onDragLeave={[Function]}
                                    onFocus={[Function]}
                                    onKeyDown={[Function]}
                                    onKeyUp={[Function]}
                                    onMouseDown={[Function]}
                                    onMouseLeave={[Function]}
                                    onMouseUp={[Function]}
                                    onTouchEnd={[Function]}
                                    onTouchMove={[Function]}
                                    onTouchStart={[Function]}
                                    tabIndex={0}
                                    type="button"
                                  >
                                    <span
                                      className="MuiButton-label"
                                    >
                                      ログイン
                                    </span>
                                    <WithStyles(memo)
                                      center={false}
                                    >
                                      <ForwardRef(TouchRipple)
                                        center={false}
                                        classes={
                                          Object {
                                            "child": "MuiTouchRipple-child",
                                            "childLeaving": "MuiTouchRipple-childLeaving",
                                            "childPulsate": "MuiTouchRipple-childPulsate",
                                            "ripple": "MuiTouchRipple-ripple",
                                            "ripplePulsate": "MuiTouchRipple-ripplePulsate",
                                            "rippleVisible": "MuiTouchRipple-rippleVisible",
                                            "root": "MuiTouchRipple-root",
                                          }
                                        }
                                      >
                                        <span
                                          className="MuiTouchRipple-root"
                                        >
                                          <TransitionGroup
                                            childFactory={[Function]}
                                            component={null}
                                            exit={true}
                                          />
                                        </span>
                                      </ForwardRef(TouchRipple)>
                                    </WithStyles(memo)>
                                  </button>
                                </ForwardRef(ButtonBase)>
                              </WithStyles(ForwardRef(ButtonBase))>
                            </ForwardRef(Button)>
                          </WithStyles(ForwardRef(Button))>
                        </div>
                      </ForwardRef(Toolbar)>
                    </WithStyles(ForwardRef(Toolbar))>
                  </Header__SpacedToolbar>
                </header>
              </ForwardRef(Paper)>
            </WithStyles(ForwardRef(Paper))>
          </ForwardRef(AppBar)>
        </WithStyles(ForwardRef(AppBar))>
      </Header>
      <Layout__ComponentWrapper>
        <main
          className="Layout__ComponentWrapper-sc-1mgn808-2 dmkdqE"
        >
          <div>
            hoge
          </div>
        </main>
      </Layout__ComponentWrapper>
      <Layout__WrapperFooter>
        <footer
          className="Layout__WrapperFooter-sc-1mgn808-1 evsPkg"
        >
          <Footer>
            <Footer__Container>
              <div
                className="Footer__Container-sc-1vgysp-0 jYsbCe"
              >
                <Footer__Urls>
                  <div
                    className="Footer__Urls-sc-1vgysp-1 sZKbU"
                  >
                    <SlashLink
                      links={
                        Array [
                          Object {
                            "rel": "noopener",
                            "text": "よくある質問",
                            "url": "http://www.google.com",
                          },
                          Object {
                            "rel": "noopener",
                            "text": "お問い合わせ",
                            "url": "http://www.google.com",
                          },
                          Object {
                            "rel": "noopener",
                            "text": "Illustrations by Storyset.",
                            "url": "https://storyset.com/work",
                          },
                        ]
                      }
                    >
                      <SlashLink__Container>
                        <div
                          className="SlashLink__Container-sc-12n56qr-0 cpzquh"
                        >
                          <SlashLink__UrlLink
                            href="http://www.google.com"
                            key="0"
                            rel="noopener"
                          >
                            <a
                              className="SlashLink__UrlLink-sc-12n56qr-1 gAUBLL"
                              href="http://www.google.com"
                              rel="noopener"
                            >
                              よくある質問
                            </a>
                          </SlashLink__UrlLink>
                          <SlashLink__UrlLink
                            href="http://www.google.com"
                            key="1"
                            rel="noopener"
                          >
                            <a
                              className="SlashLink__UrlLink-sc-12n56qr-1 gAUBLL"
                              href="http://www.google.com"
                              rel="noopener"
                            >
                              お問い合わせ
                            </a>
                          </SlashLink__UrlLink>
                          <SlashLink__UrlLink
                            href="https://storyset.com/work"
                            key="2"
                            rel="noopener"
                          >
                            <a
                              className="SlashLink__UrlLink-sc-12n56qr-1 gAUBLL"
                              href="https://storyset.com/work"
                              rel="noopener"
                            >
                              Illustrations by Storyset.
                            </a>
                          </SlashLink__UrlLink>
                        </div>
                      </SlashLink__Container>
                    </SlashLink>
                  </div>
                </Footer__Urls>
                <Footer__CopyRight
                  variant="caption"
                >
                  <WithStyles(ForwardRef(Typography))
                    className="Footer__CopyRight-sc-1vgysp-2 jQkOzQ"
                    variant="caption"
                  >
                    <ForwardRef(Typography)
                      className="Footer__CopyRight-sc-1vgysp-2 jQkOzQ"
                      classes={
                        Object {
                          "alignCenter": "MuiTypography-alignCenter",
                          "alignJustify": "MuiTypography-alignJustify",
                          "alignLeft": "MuiTypography-alignLeft",
                          "alignRight": "MuiTypography-alignRight",
                          "body1": "MuiTypography-body1",
                          "body2": "MuiTypography-body2",
                          "button": "MuiTypography-button",
                          "caption": "MuiTypography-caption",
                          "colorError": "MuiTypography-colorError",
                          "colorInherit": "MuiTypography-colorInherit",
                          "colorPrimary": "MuiTypography-colorPrimary",
                          "colorSecondary": "MuiTypography-colorSecondary",
                          "colorTextPrimary": "MuiTypography-colorTextPrimary",
                          "colorTextSecondary": "MuiTypography-colorTextSecondary",
                          "displayBlock": "MuiTypography-displayBlock",
                          "displayInline": "MuiTypography-displayInline",
                          "gutterBottom": "MuiTypography-gutterBottom",
                          "h1": "MuiTypography-h1",
                          "h2": "MuiTypography-h2",
                          "h3": "MuiTypography-h3",
                          "h4": "MuiTypography-h4",
                          "h5": "MuiTypography-h5",
                          "h6": "MuiTypography-h6",
                          "noWrap": "MuiTypography-noWrap",
                          "overline": "MuiTypography-overline",
                          "paragraph": "MuiTypography-paragraph",
                          "root": "MuiTypography-root",
                          "srOnly": "MuiTypography-srOnly",
                          "subtitle1": "MuiTypography-subtitle1",
                          "subtitle2": "MuiTypography-subtitle2",
                        }
                      }
                      variant="caption"
                    >
                      <span
                        className="MuiTypography-root Footer__CopyRight-sc-1vgysp-2 jQkOzQ MuiTypography-caption"
                      >
                        ©2021 YPS Team.
                      </span>
                    </ForwardRef(Typography)>
                  </WithStyles(ForwardRef(Typography))>
                </Footer__CopyRight>
              </div>
            </Footer__Container>
          </Footer>
        </footer>
      </Layout__WrapperFooter>
    </div>
  </Layout__Container>
</Layout>
`;
